test/es-module/test-esm-invalid-data-urls.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
I would sort of like to see successful test cases too (not just errors) if possible.

Not sure if that is asking for too much or if it is already tested elsewhere.

Reply:
Successful `import` is already covered by a lot of test files, E.G.: `test/es-module/test-esm-data-urls.js` or `test/es-module/test-esm-basic-imports.mjs`.